### PR TITLE
Core/Scripts: zone_terokkar_forest cleanup

### PR DESCRIPTION
**Pull request by ghost**
**Changes proposed**:
- moved spell and quest IDs into enum
- replaced hardcoded text with DB gossip
- restored correct gossip option order for Skull Pile

**Target branch(es)**: 3.3.5 & 6.x

**Issues addressed**: General cleanup of the `zone_terokkar_forest.cpp` script file

**Tests performed**: (Does it build? Tested in-game?) Yes.

**Known issues and TODO list**:
Several (if not all) of these individual scripts could likely be converted into SAI, but the reason I have not done it is because I do not know how to set up timed action scripts or escort scripts in SAI, which will take me weeks to learn while not making any improvements to these scripts.

### CODE REVIEW

File: sql/updates/world/2016_02_29_29_world.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-- Create gossip_menu_option 7731,7732 for NPC 18588 Floon in Terokkar Forest
+DELETE FROM `gossip_menu_option` WHERE `menu_id` IN (7731,7732) AND `id` = 0;
+INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`) VALUES
+(7731,0,0,"He wants his "golds." Pay up or die... again...",15564,1,1,0,0,0,0,'',0),
```
Comment (MitchesD) [inline sql/updates/world/2016_02_29_29_world.sql]: this is wrong, you must escape this "golds" otherwise it wont import and travis will still fail

**Comment by jackpoz** [comment]
> By analyzing the blame information on this pull request, we identified @click, @Gacko and @jackpoz to be potential reviewers

can we kill this useless annoying bot ? Otherwise I'll start ignoring every @jackpoz mention I get in the whole TC repo.

File: sql/updates/world/2016_02_29_29_world.sql (hunk 1 above)
Comment (Kittnz) [inline sql/updates/world/2016_02_29_29_world.sql]: ' '

**Comment by Kittnz** [comment]
It can, https://github.com/facebook/mention-bot but i think the repo owner needs to change this.

**Comment by Shauren** [comment]
disabled that bot

File: sql/updates/world/2016_02_29_29_world.sql (hunk 1 above)
Comment (tkrokli) [inline sql/updates/world/2016_02_29_29_world.sql]: Aha, right. I did not see that before now. Thanks for pointing this out to me.

**Comment by tkrokli** [comment]
Sorry. I lost my work in some overwrite operation when Git Extensions forced me to merge pull instead of pushing my local changes to my own fork. I now have to do it all over again from scratch.

**Pull request closed without merging**